code/modules/reagents/Chemistry-Reagents/Consumable-Reagents/Food-Reagents.dm

```diff
@@ -410,19 +410,412 @@ datum/reagent/consumable/eggyolk
+	name = "cola"
+	id = "cola"
+	result = "cola"
+	required_reagents = list("carbon" = 1, "oxygen" = 1, "water" = 1, "sugar" = 1)
+	result_amount = 4
+	mix_message = "The mixture begins to fizz."
+
+/datum/reagent/honey
+	name = "Honey"
+	id = "honey"
+	description = "A sweet substance produced by bees through partial digestion. Bee barf."
+	reagent_state = LIQUID
+	color = "#CFCF1F"
+
+datum/reagent/honey/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	M.reagents.add_reagent("sugar", 0.8)
+	..()
+	return
+
+
+/datum/reagent/chocolate
+	name = "Chocolate"
+	id = "chocolate"
+	description = "Chocolate is a delightful product derived from the seeds of the theobroma cacao tree."
+	reagent_state = LIQUID
+	color = "#2E2418"
+
+datum/reagent/chocolate/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	M.reagents.add_reagent("sugar", 0.8)
+	..()
+	return
+
+/datum/reagent/porktonium
+	name = "Porktonium"
+	id = "porktonium"
+	description = "A highly-radioactive pork byproduct first discovered in hotdogs."
+	reagent_state = LIQUID
+	color = "#AB5D5D"
+	metabolization_rate = 0.2
+	overdose_threshold = 125
+
+datum/reagent/porktonium/overdose_process(var/mob/living/M as mob)
+	if(volume > 125)
+		if(prob(8))
+			M.reagents.add_reagent("cyanide", 10)
+			M.reagents.add_reagent("radium", 15)
+	..()
+	return
+
+/datum/reagent/fungus
+	name = "Space fungus"
+	id = "fungus"
+	description = "Scrapings of some unknown fungus found growing on the station walls."
+	reagent_state = LIQUID
+	color = "#C87D28"
+
+datum/reagent/fungus/reaction_mob(var/mob/M, var/method=TOUCH, var/volume)
+	if(!istype(M, /mob/living))
+		return
+	if(method == INGEST)
+		M << "<span class = 'danger'>Yuck!</span>"
+
+/datum/reagent/chicken_soup
+	name = "Chicken soup"
+	id = "chicken_soup"
+	description = "An old household remedy for mild illnesses."
+	reagent_state = LIQUID
+	color = "#B4B400"
+
+/datum/reagent/chicken_soup/on_mob_life(var/mob/living/M as mob)
+	M.nutrition += 2
+	..()
+	return
+
+/datum/reagent/msg
+	name = "Monosodium glutamate"
+	id = "msg"
+	description = "Monosodium Glutamate is a sodium salt known chiefly for its use as a controversial flavor enhancer."
+	reagent_state = LIQUID
+	color = "#F5F5F5"
+
+datum/reagent/msg/reaction_mob(var/mob/M, var/method=TOUCH, var/volume)
+	if(!istype(M, /mob/living))
+		return
+	if(method == INGEST)
+		M << "<span class = 'notice'>That tasted amazing!</span>"
+
+
+/datum/reagent/msg/on_mob_life(var/mob/living/M as mob)
+	if(prob(1))
+		M.Stun(rand(4,10))
+		M << "<span class='warning'>A horrible migraine overpowers you.</span>"
+	..()
+	return
+
+/datum/reagent/cheese
+	name = "Cheese"
+	id = "cheese"
+	description = "Some cheese."
+	reagent_state = SOLID
+	color = "#FFFF00"
+
+/datum/chemical_reaction/cheese
+	name = "weird_cheese"
+	id = "weird_cheese"
+	result = "weird_cheese"
+	required_reagents = list("vomit" = 1, "milk" = 1)
+	result_amount = 1
+	mix_message = "The mixture curdles up."
```

Comment:
Wait the only failure here is the id and result. Weird_cheese has the recipie of green-vomit and milk. Mmm, because cheese is made of fucking vomit.

Cheese should be made by fucking grinding actual cheese you dip